### PR TITLE
WT-8952 list time groups list api

### PR DESCRIPTION
**Pull request by sukolenvo**

### CODE REVIEW

File: spec/openapi.yaml [hunk 1]
```diff
@@ -179,7 +179,7 @@ paths:
   /postedtime:
     get:
       operationId: fetch-posted-time
-      summary: Returns posted time for the team.
+      summary: Returns posted time with pending status.
       description: >
         This is a long polling call (an alternative to using the webhook mechanism).
```
Comment (vyshane) [inline spec/openapi.yaml:185]:
```suggestion
        Use this endpoint to fetch posted time for processing.
        This is a long polling call (an alternative to using the webhook mechanism).
```

File: spec/openapi.yaml [hunk 2]
```diff
@@ -212,6 +212,46 @@ paths:
...
+        Returns all time groups for team in descending order of submission time starting from now. This is not a long polling endpoint.
+      tags:
+        - Posted Time List
+      parameters:
+        - in: query
+          name: limit
+          schema:
+            type: integer
+          required: false
+          description: Maximum number of items in return. Should be from 1 to 500. Default is 100.
```
Comment (vyshane) [inline spec/openapi.yaml]:
```suggestion
          description: Maximum number of items to return. Should be between 1 and 500. Default is 100.
```

File: spec/openapi.yaml [hunk 3]
```diff
@@ -720,6 +768,7 @@ components:
           description: The duration of the activity, in seconds.
         submittedDate:
           type: long
+          deprecated: true
```
Comment (vyshane) [inline spec/openapi.yaml:772]: @pfilippi24 this is being deprecated in favour of `submissionTime` on the time group.

**Review by vyshane — APPROVED** [review]
Added a couple of small suggestions. LGTM.

**Review by pfilippi24 — APPROVED** [review]
LGTM

**Pull request merged**